test/lint.js

```diff
@@ -0,0 +1,145 @@
+'use strict';
+
+let assert = require('assert');
+let emu = require('../lib/ecmarkup');
+
+function alg([before, after], marker) {
+  return `<emu-alg>${before}${marker}${after}</emu-alg>`;
+}
+
+let M = '@@@'; // Marks the location of the error
+
+describe('linting', function () {
+  describe('grammar sanity', function () {
```

Comment:
```suggestion
  describe('grammar validity', function () {
```

may be a better choice of words.

Reply:
Done.